Detections/AzureDiagnostics/KeyVaultSensitiveOperations.yaml

```diff
@@ -36,4 +36,5 @@ query: |
   | where ResourceType == "VAULTS" and ResultType == "Success" 
   | where OperationName in (SensitiveOperationList)  
   | project TimeGenerated, ResourceType, ResultType, Resource, OperationName, id_s, CallerIPAddress, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s
+  | summarize count(), TimeGeneratedMax=arg_max(TimeGenerated,*), TimeTriggered=makelist(TimeGenerated),OperationNameList=make_set(OperationName), RequestURLList=make_set(requestUri_s), CallerIPList = make_set(CallerIPAddress),  CallerIPMax= arg_max(CallerIPAddress,*) by ResourceType, ResultType, Resource, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, clientInfo_s
```

Comment:
Are you able to remove the Project above this and just go with the summarize?  Also, lets label what we are counting so something like CallerCount = count().